test/pull-weird.js

```diff
@@ -11,7 +11,7 @@ var pull = require('pull-stream')
 tape('aborts pull-weird correctly', function (t) {
 
   t.plan(2)
-  var ps = new PacketStream({})
+  var ps = PacketStream({})
```

Comment:
Tests were failing on this line because PacketStream was [never meant to be called as a constructor](https://github.com/ssbc/packet-stream/blob/c3b4d75794dc6e89ea84d2c9fcb309259de723f6/index.js#L7-L9) but now that we [recently changed it to an arrow function](https://github.com/ssbc/packet-stream/blob/bb0493dd0eb6454ac675df78dfed20e994a4881f/index.js#L230) it strictly **cannot** be used as a constructor.

Reply:
I grepped the whole source of browser-core to see if we had more of these new PacketStream and this test was the only usage ;-)